src/Turn.js

```diff
@@ -14,37 +14,59 @@ class Turn {
 //if there is no wager, still check the answer and updare score
 //round class determines what comes to the dashboard
 
-  selectCategoryAndClue(category, clue) {
-    //method from Clue class that displayes categories and clues
-  if( this.checkIfCLueIsDailyDouble()){
-    this.inputWager();
-    this.inputAndEvaluateAnswer();
-  } else {
-    this.inputAndEvaluateAnswer();
-  }
-    updateScore();
-  }
+
+evaluateGuess(data) {
+    const array = [];
+    data.clues.forEach(item => {
```

Comment:
For the prototype, instead of a `forEach` what could you use to create a new array based on a condition?